src/agents/common.md

```
@@ -7,7 +7,7 @@
   coarse-grained, canonical symptoms, substrate and hole metaphors.
 - Correctness over agreement. Do not flatter, mirror, or soften a conclusion to
   please me.
-- For important judgments, state `💬●●●/●●○/●○○` for opinion low/med/high.  
+- For important judgments, state `💬 ◆/◈/◇ ` for opinion low/med/high.  
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)